### PR TITLE
Update test cases

### PR DESCRIPTION
**Pull request by Yasara123**

### CODE REVIEW

File: component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java [hunk 1]
```diff
@@ -758,6 +761,12 @@ public void fileSinkTest() throws InterruptedException {
 
         Thread.sleep(1000);
         siddhiAppRuntime.shutdown();
+        File sinkRoot = new File(sinkUri);
+        try {
+            FileUtils.deleteDirectory(sinkRoot);
+        } catch (IOException e) {
+            throw new TestException("Failed to delete files in due to " + e.getMessage(), e);
```
Comment (dilini-muthumala) [inline component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java]: Better log the sinkUri too.

File: component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java [hunk 2]
```diff
@@ -810,13 +823,19 @@ public void fileSinkTestGroup() throws InterruptedException {
...
             AssertJUnit.fail(sinkUri + " is not a directory.");
         }
 
         Thread.sleep(1000);
         siddhiAppRuntime.shutdown();
+        File sinkRoot = new File(sinkUri);
+        try {
+            FileUtils.deleteDirectory(sinkRoot);
+        } catch (IOException e) {
+            throw new TestException("Failed to delete files in due to " + e.getMessage(), e);
```
Comment (dilini-muthumala) [inline component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java]: log the sinkUri too

File: component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextDefaultSinkMapperTestCase.java [hunk 3]
```diff
@@ -586,6 +591,12 @@ public void fileSinkTestSingle() throws InterruptedException {
 
         Thread.sleep(1000);
         siddhiAppRuntime.shutdown();
+        File sinkRoot = new File(sinkUri);
+        try {
+            FileUtils.deleteDirectory(sinkRoot);
+        } catch (IOException e) {
+            throw new TestException("Failed to delete files in due to " + e.getMessage(), e);
```
Comment (dilini-muthumala) [inline component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextDefaultSinkMapperTestCase.java]: log the sinkUri too

File: component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java (hunk 1 above)
Comment (Yasara123) [reply component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java]: Fixed in cdb7b86019f370bcb90c545043b2ad205e8e98df

File: component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java (hunk 2 above)
Comment (Yasara123) [reply component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java]: Fixed in cdb7b86019f370bcb90c545043b2ad205e8e98df

**Pull request merged**